src/main/java/edu/kis/vh/nursery/utility/IntArrayStack.java

```diff
@@ -1,42 +1,46 @@
-package edu.kis.vh.nursery;
+package edu.kis.vh.nursery.utility;
 
-public class IntArrayStack {
+public class IntArrayStack implements StackImplementation {
 
-    private static final int IS_EMPTY_INDICATOR = -1;
-
-    private static final int CAPACITY = 12;
+    private static final int CAPACITY = 13;
```

Comment:
Changing IS_EMPTY_INDICATOR value requires such fixes? It does not make sense. Incorrect fix.